integration/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -126,9 +129,11 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
               .setName("Tachyon Master Executor")
               .setSource("master")
               .setExecutorId(Protos.ExecutorID.newBuilder().setValue("master"))
-              .setCommand(
-                  Protos.CommandInfo.newBuilder().setValue(PathUtils
-                      .concatPath(tachyonHome, "integration", "bin", "tachyon-master.sh")));
+              .setCommand(Protos.CommandInfo.newBuilder().setValue(
+                  "export JAVA_HOME=" + conf.get(Constants.JRE_VERSION) 
```

Comment:
Should we create Constant.JAVA_HOME instead? Currently, the default value will produce JAVA_HOME=jre1.7.0_76, which won't ever work. If the default is not expected to work, I would not use it.